CHECKLIST

```diff
@@ -24,6 +24,8 @@ The topics can be an empty list, and the difficulty level is from 1 to 10, and c
 ]
 ```
 
+The order of the exercises is decided by the order in this "exercises" array. Try to make them roughly increasing in difficulty, but don't worry too much—we can always adjust the order later, as we get feedback from people doing the exercises.
```

Comment:
May I suggest a little word smithing?

`Problems are delivered to users in the same order that they are listed in this "exercises" array. Try to order them by increasing difficulty, but don't worry too much—it can always be adjusted later.`


`, as we get feedback from people doing the exercises.` - We don't have any specific mechanism for gathering this feedback that I know of.